tests/Fixer/Strict/DeclareStrictTypesFixerTest.php

```diff
@@ -141,11 +155,11 @@ public function provideMessyWhitespacesCases()
     {
         return [
             [
-                "<?php declare(strict_types=1);\r\nphpinfo();",
+                "<?php\r\ndeclare(strict_types=1);\r\n\tphpinfo();",
```

Comment:
> if we claim to fix in one way (eg "same line"), and then out of the blue redefine the rule to fix code another way (eg "new line")

The only thing the current fixer says is
> Force strict types declaration in all files.

Not sure that really makes any guarantees about position, but nevertheless I'm happy to change to insert on same line – I was simply unaware of the compatibility requirement and figured this was a better option for working with the proposed PSR standard concerning it.

I shall change the insertion fixer to insert on a new line, and the positioning fixer will stay as is.

---

> > Should I bug this then?
> 
> no, i have no idea what is `dev-feature/braces/position_after_return_type_hint` version you had installed.

`dev-feature/braces/position_after_return_type_hint` is my branch based on `master` with two unrelated commits. The cron failed when I rebased my commits on that branch (so something at your end changed).
As for whether it's still like that, I'd have to check. I'd also have to check what it did before. Looks as if it is a BC-break if this is indeed a change from a release version, per what has been laid out here (in-fact maybe even worse so because it modifies code that has already gone through the fixer).

Happy to look into it at some point, just trying to give a heads up on a possible bug here really, since this seems to be an important consideration?